src/demo/app/app.routing.ts

```diff
@@ -3,6 +3,7 @@ import { Routes, RouterModule } from '@angular/router';
 
 import { TocComponent } from './toc.component';
 import { AdvancedConfirmDemoComponent } from './fancy-form/advanced-confirm-demo.component';
+import { ButtonDemoComponent } from './fancy-form/button-demo.component';
```

Comment:
dup import name on line 21 preventing `npm start`! 
I assume that we should be using `fancy-form/button-demo.component`  (and should delete `global-css/button-demo.component` and its import)?